views/js/app.jsx

```diff
@@ -111,34 +112,45 @@ class LoginHome extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
-        reps: []
+      user_guid: null,
+      reps: []
```

Comment:
I added this `user_guid` property because this was previously a part of `this.state.reps` (I think it came as part of the initial get request), but if we're not using it, I can just remove it entirely.